src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -404,6 +409,14 @@ public Action getProjectAction(AbstractProject<?, ?> project) {
     return new SonarMarkerAction();
   }
 
+  protected Object readResolve() {
+    // Migrate old field to new field
+    if (sonarRunnerName != null) {
+      sonarScannerName = sonarRunnerName;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "sonarRunnerName"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)